sigmf-spec.md

```diff
@@ -336,43 +338,58 @@ about the Recording itself.
-| `geolocation`   | false    | GeoJSON `point` object | The location of the Recording system.|
-| `extensions`    | false    | array   | A list of JSON Objects describing extensions used by this Recording.|
-| `collection`    | false    | string  | The base filename of a `collection` with which this Recording is associated.|
+| name             | required | type    | description      |
+| ---------------- | -------- | --------| -----------------|
+| `datatype`       | true     | string  | The SigMF Dataset format of the stored samples in the dataset file.|
+| `sample_rate`    | false    | double  | The sample rate of the signal in samples per second.|
+| `version`        | true     | string  | The version of the SigMF specification used to create the Metadata file.|
+| `num_channels`   | false    | uint    | Total number of interleaved channels in the dataset file. If omitted, this defaults to one.|
+| `sha512`         | false    | string  | The SHA512 hash of the Dataset file associated with the SigMF file.|
+| `offset`         | false    | uint    | The index number of the first sample in the Dataset. If not provided, this value defaults to zero. Typically used when a Recording is split over multiple files. All sample indices in SigMF are absolute, and so all other indices referenced in metadata for this recording SHOULD be greater than or equal to this value.|
+| `description`    | false    | string  | A text description of the SigMF Recording.|
+| `author`         | false    | string  | A text identifier for the author potentially including name, handle, email, and/or other ID like Amateur Call Sign. For example "Bruce Wayne <bruce@waynetech.com>" or "Bruce (K3X)".|
+| `meta_doi`       | false    | string  | The registered DOI (ISO 26324) for a Recording's Metadata file.|
+| `data_doi`       | false    | string  | The registered DOI (ISO 26324) for a Recording's Dataset file.|
+| `recorder`       | false    | string  | The name of the software used to make this SigMF Recording.|
+| `license`        | false    | string  | A URL for the license document under which the Recording is offered.|
+| `hw`             | false    | string  | A text description of the hardware used to make the Recording.|
+| `dataset`        | false    | string  | The full filename of the dataset file this Metadata file describes.|
+| `trailing_bytes` | false    | uint    | The number of bytes to ignore at the end of a Non-Conforming Dataset file.|
+| `virtual_dataset`| false    | bool    | Indicates that the dataset is virtual (SigMF Recording is metadata only).|
+| `geolocation`    | false    | GeoJSON `point` object | The location of the Recording system.|
+| `extensions`     | false    | array   | A list of JSON Objects describing extensions used by this Recording.|
+| `collection`     | false    | string  | The base filename of a `collection` with which this Recording is associated.|
```

Comment:
I can't tell what happened, here. Is there a whitespace change that got added in to this table?

Reply:
yes - `virtual_dataset` is one character longer than the previous longest field so to preserve alignment these were all shifted one space.